File: docs/vsto/codesnippet/CSharp/trin_wordcontentcontrolreference/checkbox.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Microsoft.VisualStudio.Tools.Applications.Runtime;
using Word = Microsoft.Office.Interop.Word;
using Office = Microsoft.Office.Core;

namespace Trin_ContentControlReference
{
    public partial class ThisDocument
    {
        private void AddCheckBoxControlAtSelection()
        {
            //<Snippet800>
            this.Paragraphs[1].Range.InsertParagraphBefore();
            this.Paragraphs[1].Range.Select();
            Microsoft.Office.Tools.Word.ContentControl checkBoxControl1 = 
                this.Controls.AddContentControl("checkBoxControl1", Word.WdContentControlType.wdContentControlCheckBox);
            
            checkBoxControl1.Checked = true;

            //</Snippet800>
        }
    }
}

File: docs/vsto/codesnippet/VisualBasic/trin_contentcontrolreference/checkbox.vb
Partial Public NotInheritable Class ThisDocument
    Private Sub AddCheckBoxControlAtSelection()
        '<Snippet800>
        Me.Paragraphs(1).Range.InsertParagraphBefore()
        Me.Paragraphs(1).Range.Select()
        Dim checkBoxControl1 As Microsoft.Office.Tools.Word.ContentControl =
            Me.Controls.AddContentControl("checkBoxControl1", Word.WdContentControlType.wdContentControlCheckBox)


        checkBoxControl1.Checked = True

        '</Snippet800>
    End Sub
End Class

